src/inc/til/rectangle.h

```diff
@@ -875,6 +883,22 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
         }
 #endif
 
+#ifdef WINRT_Microsoft_Terminal_Core_H
+        operator winrt::Microsoft::Terminal::Core::Padding() const noexcept
+        {
+            winrt::Microsoft::Terminal::Core::Padding ret;
+            THROW_HR_IF(E_ABORT, !base::MakeCheckedNum(left()).AssignIfValid(&ret.Left));

```

Comment:
now a padding isn't really a rectangle, and the conflation scares me. I know til::margins would be a useless auxiliary class, though.

@miniksa what do you think about this?

Reply:
meh okay!

Reply:
> meh okay!

You've captured exactly how I feel about this. I get that it isn't a rectangle as would be inscribed from the middle... but I feel like padding is rectangularish enough that I'm fine with this if it makes life easier.